docs/product/stats/index.mdx

```diff
@@ -10,11 +10,87 @@ The "Usage" tab shows your organization’s event and attachments usage, while a
+| Invalid Transaction             | The event is parseable but semantically invalid. This should only happen with transaction events.                       |
+| Missing Minidump Upload         | A minidump file was missing for the minidump endpoint.                                                                  |
+| Multi Project Id                | The project id in the URL does not match the one specified for the public key.                                          |
+| No Data                         | The store request was missing an event payload.                                                                         |
+| No Event Payload                | An event envelope was submitted but no payload could be extracted.                                                      |
+| Payload                         | Parsing the event JSON payload failed due to a syntax error.                                                            |
+| Process Unreal                  | Symbolic failed to extract an Unreal Crash report from a request sent to the Unreal endpoint.                           |
+| Profiling                       | Profiling related dropped reasons.                                                                                      |
+| Project Id                      | There was no valid project id in the request or the required project does not exist.                                    |
+| Project State                   | A project state returned by the upstream could not be parsed.                                                           |
+| Project State Pii               | A project state returned by the upstream contained datascrubbing settings that could not be converted to PII config.    |
+| Rate Limited                    | Exceeded allowed event rate.                                                                                            |
+| Spike Protection                | Activated to protect from any sudden quota-depleting spike.                                                             |
+| Security Report                 | Triggered to prevent any sudden spikes that could deplete the quota.                                                    |
+| Security Report Time            | The security report was not recognized due to missing data.                                                             |
+| Timestamp                       | The timestamp of an event was required for processing and either missing out of the supported time range for ingestion. |
+| Too Large                       | The event payload exceeds the maximum size limit for the respective endpoint.                                           |
+| Transaction Sampled             | The envelope, which contained only a transaction, was dropped by the dynamic sampling rules.                            |
```

Comment:
@jan-auer, could you please confirm that the information listed here and in the table above is correct?